server/services/unusedDeductionsService.ts

```diff
@@ -82,7 +89,11 @@ export default class UnusedDeductionsService {
 
       return ['UNKNOWN', adjustments]
     } catch {
-      return ['UNKNOWN', adjustments]
+      if (this.getTotalUnusedRemand(adjustments) > 0) {
```

Comment:
I think this should only be for DPS unused deductions, not if the unused deductions were entered in NOMIS.